.chloggen/ottl_function_changes.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/ottl
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Changed replacement_pattern (replacement) string to be a path expression to a string telemetry field or a literal string
```

Comment:
I think saying something along the lines of "change replacement functions to accept a path expression as a replacement" and listing the affected functions in the `subtext` key would be good here.

Reply:
sounds good!, made this change